pilot/docker/Dockerfile.pilot

```diff
@@ -17,6 +17,15 @@ WORKDIR /
 # hadolint ignore=DL3006
 FROM ${BASE_DISTRIBUTION}
 
+RUN sudo apt-get update && sudo apt-get install -y gnupg && \
+    sudo apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys 4D64390375060AA4 && \
+    echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list && \
+    curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/Release.key | sudo apt-key add - && \
+    sudo apt-get update && \
+    sudo apt-get -y upgrade && \
+    sudo apt-get -y install podman
+
 COPY pilot-discovery /usr/local/bin/
+COPY mec /usr/local/bin/
```

Comment:
Why do you need mec in pilot image?

Reply:
oops, didn't mean to push this change. I adjusted the pilot image to include mec for quick testing